docs/source/get_started/quick_start.md

```diff
@@ -0,0 +1,272 @@
+
+```python
+builder.pw.parameters['SYSTEM']['nbnd'] = 15
+builder
+
+```
+
+::::{note}
+
+The protocols make it easier to get a starting set of inputs.
+However, you can also start from an empty `builder`:
+
+```python
+builder = PwBaseWorkChain.get_builder()
+```
+
+:::{dropdown} Minimal working example
+```python
+pseudo_family = orm.load_group('SSSP/1.3/PBEsol/efficiency')
+
+builder = PwBaseWorkChain.get_builder()
+builder.kpoints_distance = 0.3
+builder.pw.code = orm.load_code('pw@localhost')
+builder.pw.structure = structure
+builder.pw.pseudos = pseudo_family.get_pseudos(structure=structure)
+builder.pw.parameters = {
+    'SYSTEM': {
+        'nbnd': 10
+    },
+    'CONTROL': {
+        'calculation': 'scf'
+    }
+}
+results = engine.run(builder)
+```
+:::
+
+Finally, you can also directly pass your inputs to the engine by preparing all of them in an `inputs` dictionary:
+
+```
+engine.run(PwBaseWorkChain, **inputs)
+```
+This is shown in the dropdown below.
+
+:::{dropdown}  Minimal working example
+```python
+pseudo_family = orm.load_group('SSSP/1.3/PBEsol/efficiency')
+
+inputs = {
+    'kpoints_distance': 0.3,
+    'pw': {
+        'code': orm.load_code('pw@localhost'),
+        'structure': structure,
+        'pseudos': pseudo_family.get_pseudos(structure=structure),
+        'parameters': {
+            'CONTROL': {
+                'calculation': 'scf'
+            }
+        }
+    }
+}
+results = engine.run(PwBaseWorkChain, **inputs)
+```
+:::
+
+::::
+
+## Submitting and inspecting processes
+
+So far, we've used the `engine.run()` function to run the `PwBaseWorkChain`.
+This is very useful for short running processes in tutorials such as this one, or while developing a workflow.
+However, if the `run` command is interrupted, so is the `pw.x` calculation we are running.
+
+Instead, we typically want to `sumbit` the process to the AiiDA _daemon_, which will take care of its execution.
+
+```python
+from aiida_quantumespresso.workflows.pw.base import PwBaseWorkChain
+
+builder = PwBaseWorkChain.get_builder_from_protocol(
+    code=code,
+    structure=structure,
+    protocol='fast'
+)
+workchain_node = engine.submit(builder)
+```
+
+:::{important}
+
+The `submit()` function returns the work chain _node_, not the results!
+This makes sense: after submitting the calculation, it hasn't been run yet, so there are no results to return.
+
+:::
+
+The AiiDA daemon is a process that can run in the background and manage your calculations and workflows.
+Let's see if it's running:
+
+:::{margin}
+**Note**: Lines starting with `!` aren’t Python.
+This is an [IPython/Jupyter shell escape](https://ipython.readthedocs.io/en/stable/interactive/reference.html#system-shell-access) — the command after `!` is run in your system shell (e.g. `bash` or `zsh`), not in Python itself.
+Alternatively, you can open a terminal and run these commands there (minus the `!`).
+:::
+
+```
+!verdi daemon status
```

Comment:
The idea is that this tutorial can be downloaded as a Jupyter notebook, in `.ipynb` format. Unfortunately, our somewhat outdated documentation setup doesn't offer that option. I'll make sure to fix this after we merge our first round of documentation PRs.